society-of-biblical-literature-fullnote-bibliography.csl

```diff
@@ -876,7 +915,33 @@
   <macro name="pages">
     <choose>
       <if type="article-journal">
-        <text variable="page" prefix=": "/>
+        <!-- Add support for individually paginated, electronic journal articles per https://sblhs2.com/2018/05/03/electronic-journals-with-individually-paginated-articles/ -->
+        <choose>
+          <if variable="number">
+            <text term="article-journal" form="verb-short" prefix=": "/>
+            <text variable="number" prefix=" " suffix=", "/>
+            <choose>
+              <if variable="locator">
```

Comment:
The only time this macro is getting called is after a negative test for locator, so this doesn't make sense here: you can remove the entire locator logic from this iteration.

Reply:
I think the update that I'll submit shortly makes this section worth keeping again. Of course, do let me know though if I'm mistaken in that.